locust/contrib/fasthttp.py

```diff
@@ -0,0 +1,383 @@
+from __future__ import absolute_import
+
+import chardet
+import re
+import six
+import socket
+from base64 import b64encode
+from six.moves.urllib.parse import urlparse, urlunparse
+from ssl import SSLError
+from timeit import default_timer
+
+if six.PY2:
+    from cookielib import CookieJar
+    class ConnectionRefusedError(Exception):
+        # ConnectionRefusedError doesn't exist in python 2, so we'll 
+        # define a dummy class to avoid a NameError
+        pass
+    str = unicode
+else:
+    from http.cookiejar import CookieJar
+
+from gevent.timeout import Timeout
+from geventhttpclient.useragent import UserAgent, CompatRequest, CompatResponse, ConnectionError
+from geventhttpclient.response import HTTPConnectionClosed
+
+from locust import events
+from locust.core import Locust
+from locust.exception import LocustError, CatchResponseError, ResponseError
+
+
+# Monkey patch geventhttpclient.useragent.CompatRequest so that Cookiejar works with Python >= 3.3
+# More info: https://github.com/requests/requests/pull/871
+CompatRequest.unverifiable = False
```

Comment:
should this only be done for python 3 in the `else` above?

Reply:
Doesn't really matter. I've also submitted a PR that fixes this is geventhttpclient, which has been merged, so we should be able to remove this altogether once they release a new version.